conf/distro/openxt-main.conf

```diff
@@ -35,10 +35,6 @@ PREFERRED_VERSION_linux-libc-headers-nativesdk ?= "${@"${PREFERRED_VERSION_linux
 
```

Comment:
This development commit needs more substance for submission :).

Can you add the ticket number to the commit message: https://openxt.atlassian.net/browse/OXT-1802?